webextensions/manifest/theme.json

```
@@ -464,7 +464,9 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": "62"
+                  "version_added": "62",
+                  "version_removed": "89",
+                  "notes": "This theme property has been deprecated in Firefox 89. See the <a href='https://blog.mozilla.org/addons/2021/04/19/changes-to-themeable-areas-of-firefox-in-version-89/'>related blogpost</a> for more details."
```

Do we need to state this is a theme property? Isn't that self-evident?
```suggestion
                  "notes": "Deprecated in Firefox 89. For details, see the <a href='https://blog.mozilla.org/addons/2021/04/19/changes-to-themeable-areas-of-firefox-in-version-89/'>Changes to themeable areas of Firefox in version 89</a> blog post ."
```

---

> Do we need to state this is a theme property? Isn't that self-evident?
+1 @rebloor I agree with you, I'll apply your edits (which does also integrate Rob suggestion in addition to yours) in the next update to this patch